radio/apps/schedules/models.py

```diff
@@ -104,11 +101,10 @@ def save(self, *args, **kwargs):
 
     @classmethod
     def get_current(cls, dt):
-        schedule_board = cls.objects.filter(
-            start_date__lte=dt, end_date__isnull=True
-        ).order_by('-start_date') | cls.objects.filter(
-            start_date__lte=dt, end_date__gte=dt
-        ).order_by('-start_date')
+        schedule_board = (
+            cls.objects.filter(start_date__lte=dt, end_date__isnull=True).order_by('-start_date') |
+            cls.objects.filter(start_date__lte=dt, end_date__gte=dt).order_by('-start_date')
+        )
```

Comment:
Make code more readable. The important part is to apply multiple filters, so the code should not spread this across multiple indentation levels, but should reflect this by intuitive reading.